mobile/src/people/people-room.tsx

```diff
@@ -4,18 +4,22 @@ import { connect, Dispatch } from 'react-redux';
 
 import { EmployeesList } from './employees-list';
 import { AppState } from '../reducers/app.reducer';
-import { EmployeeMap } from '../reducers/organization/employees.reducer';
+import { EmployeeMap, EmployeesStore } from '../reducers/organization/employees.reducer';
 import { Employee } from '../reducers/organization/employee.model';
 import { openEmployeeDetailsAction } from '../employee-details/employee-details-dispatcher';
 
 interface PeopleRoomProps {
-    employeesMap: EmployeeMap;
+    employees: EmployeesStore;
     userEmployee: Employee;
+    employeesPredicate: (employee: Employee) => boolean;
 }
 
 const mapStateToProps = (state: AppState): PeopleRoomProps => ({
-    employeesMap: state.organization.employees.employeesById,
-    userEmployee: state.userInfo.employee
+    employees: state.organization.employees,
+    userEmployee: state.userInfo.employee,
+    employeesPredicate: (employee: Employee) => {
```

Comment:
do you realize that this statement is called on each state change? and each state change returns a new function. Just make it a class method!

Reply:
ok

Reply:
addressed